workspaces/analytics/plugins/analytics-module-matomo/catalog-info.yaml

```diff
@@ -0,0 +1,46 @@
+# https://backstage.io/docs/features/software-catalog/descriptor-format#kind-component
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: backstage-community-analytics-matomo
+  title: 'Analytics Module: Matomo'
+  description: 'Analytics Module: Matomo'
+  annotations:
+    github.com/project-slug: backstage-community/backstage-plugins
+    github.com/team-slug: backstage-community/maintainers-plugins
+    sonarqube.org/project-key: backstage-community_backstage-plugins
```

Comment:
Looks like there might be some consistency problems? For the ACR plugin you have:

```
  annotations:
    github.com/project-slug: backstage/community-plugins
    github.com/team-slug: backstage/community-plugins-maintainers
    sonarqube.org/project-key: backstage-community_plugins
```
Not certain if this was intentional or not.

Reply:
Looks like none of them exists. In some cases I just moved the annotations. I will double-check and remove it if I couldn't find a (public) sonarqube project :laughing:

Reply:
Verified that none of them exist:

```
find . -iname catalog-info.yaml | xargs grep -h 'sonarqube.org/project-key' | sort | uniq -c | sort -nr
     22     sonarqube.org/project-key: backstage-community_plugins
      5     sonarqube.org/project-key: backstage-community_backstage-plugins
      2     sonarqube.org/project-key: backstage_community-plugins
      2     sonarqube.org/project-key: backstage/community-plugins
```

I will remove all `sonarqube.org/project-key` annotations.